assets/data-port/import.js

```diff
@@ -4,16 +4,16 @@ import { render, useLayoutEffect } from '@wordpress/element';
 import { useSenseiColorTheme } from '../react-hooks/use-sensei-color-theme';
 import { DataPortStepper } from './stepper';
 import registerImportStore from './import/data';
-import { Spinner } from '@woocommerce/components';
 import { Notice } from '@wordpress/components';
+import '../shared/data/api-fetch-preloaded-once';
 
 registerImportStore();
 
 /**
  * Sensei import page.
  */
 const SenseiImportPage = () => {
-	const { isFetching, error, navigationSteps } = useSelect( ( select ) => {
+	const { error, navigationSteps } = useSelect( ( select ) => {
```

Comment:
Removed in f3466ea. I left `isFetching` in the store for now, but it can probably be removed later if everything is working out with this as the spinner here was its only usage.